CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/state/AnalyticViewState.java

```diff
@@ -90,17 +98,55 @@ public void clearAnalyticQuestions() {
     }
 
     /**
-     * Check the currently selected Question index of plugins for other plugins 
+     * Check the currently selected Question index of plugins for other plugins
      * matching the selected category
+     *
      * @param currentCategory the currently selected plugin category to remove
      * from
      */
     public void removePluginsMatchingCategory(String currentCategory) {
         if (!activeSelectablePlugins.isEmpty()) {
-            activeSelectablePlugins.get(currentAnalyticQuestionIndex).removeIf(plugin -> (
-                plugin.getPlugin().getClass().getAnnotation(AnalyticInfo.class).analyticCategory().equals(currentCategory)
-                )
+            activeSelectablePlugins.get(currentAnalyticQuestionIndex).removeIf(plugin -> (plugin.getPlugin().getClass().getAnnotation(AnalyticInfo.class).analyticCategory().equals(currentCategory))
             );
         }
     }
+
+    /**
+     * Attribute provider for attributes specific to the Analytic View.
+     *
+     * @author cygnus_x-1
+     */
+    @ServiceProvider(service = SchemaConcept.class)
+    public static class AnalyticViewConcept extends SchemaConcept {
+
+        @Override
+        public String getName() {
+            return "Analytic View";
+        }
+
+        @Override
+        public Set<Class<? extends SchemaConcept>> getParents() {
+            final Set<Class<? extends SchemaConcept>> parentSet = new HashSet<>();
+            parentSet.add(ConstellationViewsConcept.class);
+            return Collections.unmodifiableSet(parentSet);
+        }
+
+        public static class MetaAttribute {
+
+            private MetaAttribute() {
+                throw new IllegalStateException("Utility class");
```

Comment:
The concept classes aren't really utility classes (at least by our definitions anyway). We usually mark the sonar rule that comes up as a false positive in these cases. I won't say it's a bad thing to include the private constructor but maybe change the string in the exception (something like "Concept class)").

Reply:
Done